<a href="https://colab.research.google.com/github/auc-biot6202-spring2022/genetic-variation/blob/main/genetic_variation_notebook.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Genetic Variation

In [1]:
import pandas as pd

In [2]:
# Reading the variation TSV file directly as a compresed (.gz) from URL
# NA12878 is person2 in the pedigree

person2 = pd.read_table('https://github.com/ahmedmoustafa/platinum-genomes/blob/main/tsv/NA12878.tsv.gz?raw=true', compression='gzip')
person2.head()

chr    pos                      id ref alt
0  chr1  14677             rs201327123   G   A
1  chr1  15922             rs375964566   A   G
2  chr1  15956             rs112448831   G   A
3  chr1  16014  rs113442401;rs75082847   C   T
4  chr1  16298   rs62636498;rs77798508   C   T

In [3]:
len(person2)

4489434

NA12878's variants table was loaded successfully, with a total of 4,489,434 variants (about 4.5 million).